src/middlewared/middlewared/plugins/account.py

```diff
@@ -588,9 +604,10 @@ async def do_delete(self, pk, options):
         await self.middleware.call('datastore.delete', 'account.bsdusers', pk)
         await self.middleware.call('service.reload', 'user')
 
-        return pk
+        return True
 
     @accepts(Int('user_id', default=None, null=True))
+    @returns(Dict('shell_choices', additional_attrs=True))
```

Comment:
For dictionaries with `additional_attrs=True` we might need something like `example={"bash": "/bin/bash", "zsh": "/usr/bin/zsh"}`, otherwise it's not much informative.